lib/organisms/gravity-form/example.html

```diff
@@ -0,0 +1 @@
+<ln-o-gravity-form id="{{id}}" token="{{token}}" data="{{data}}"></ln-o-gravity-form>
```

Comment:
Wasn't sure how you were passing data around so I put that there as a placeholder for the minute, if the token is stored in local storage in some way and you can retrieve by ID, then no, we don't need it to be passed in :)

Once I've gone over the GF Web API and stubbed it out in the .service.js I'll have a better idea of what's going on I expect.